@nav-frontend/react-accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { forwardRef, useEffect, useRef, useState } from "react";
+import cls from "classnames";
+import {
+  UnmountClosed,
+  Collapse,
+  CollapseProps,
+  CollapseCallbackArgs,
+} from "react-collapse";
+
+import { Expand } from "@nav-frontend/icons";
+import "@nav-frontend/accordion-styles";
+import { guid } from "nav-frontend-js-utils";
+
+interface AccordionProps {
+  children: React.ReactNode;
+  title: React.ReactNode;
+  open?: boolean;
+  onClick?: (event: React.SyntheticEvent<HTMLButtonElement>) => void;
+  className?: string;
+  renderContentWhenClosed?: boolean;
+  collapseProps?: Partial<CollapseProps>;
+  buttonId?: string;
```

Comment:
Hva er use-casen for å la bruker bestemme buttonId?

Reply:
Usikker, fant ingen bruk av den i navikt så fjernet den